client/components/checkbox/checkbox.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>CodePen - box_new</title>
+    <link rel="stylesheet" href="./checkbox.css">
+
+</head>
+
+<body>
+    <!-- partial:index.partial.html -->
+    <label class="container">
+  <input class="checkbox" type="checkbox" checked="checked"/>
+  <span class="checkbox-new"></span> 
+</label>
+    <!-- partial -->
+
+</body>
+
```

Comment:
We need checkbox with label for Registration form and New password form.
It would be nice if you add checkbox with label

Reply:
it is with lable, no?